lib/chef-dk/command/verify.rb

```diff
@@ -141,6 +147,35 @@ def components
           # ------------
           failures = []
 
+          # fail the verify if we have more than one version of chef-provisioning or any of its drivers.
```

Comment:
Hard error in my opinion--one big reason to have this is to fail out a build if we have multiple versions.